karma.conf.js

```diff
@@ -33,8 +33,8 @@ module.exports = function(config) {
     exclude: [],
     // Pre-process matching files before serving them to the browser.
     preprocessors: {
-      'client/*.js': preprocessorOptions,
       'client/**/*.js': preprocessorOptions,
+      'client/**/**/*.js': preprocessorOptions,
```

Comment:
I think you don't need this line, for the same reason. Could you please verify that, and remove this if so?